CHANGES.md

```diff
@@ -5,6 +5,7 @@ CKEditor 4 Changelog
 
 Fixed Issues:
 
+* [#1824](https://github.com/ckeditor/ckeditor-dev/issues/1824): Fixed: [`Autolink`](https://ckeditor.com/cke4/addon/autolink) plugin doesn't work without [`Link`](https://ckeditor.com/cke4/addon/link) plugin.
```

Comment:
In changelog we use backticks for code, but not for plugin names, so we should remove them from this entry. Also, repository name in the link needs to be updated (ckeditor-dev -> ckeditor4).

Reply:
Removed backticks and update repo address: https://github.com/ckeditor/ckeditor4/pull/2518/commits/9b9cb89b5693a11c71747e3ff7c34f0fbfc5bc43

Also added an entry about fixing #2208 issue.